### PR TITLE
move lesson from module2, adapt examples

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,244 @@
...
+
+```plaintext
+I'm a cat!
+```
+
+As we can see, `myAnimal` object is an object of type `Cat`, not of type
+`Animal` That's because there is a difference between the type at compile-time
+and the type at runtime. At compile time, meaning when the code was written, the
+type of the `myAnimal` variable was defined as `Animal`. However, when the
+program runs, a `Cat` data type is assigned to `myAnimal` variable instead. So
```
Comment (kcleland0818) [inline README.md]: Missing "the" between "to" and `myAnimal`

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ As we can see, `myAnimal` object is an object of type `Cat`, not of type
 `Animal` That's because there is a difference between the type at compile-time
```
Comment (kcleland0818) [inline README.md]: Oops. I found another one. This might have been from me... but we're missing a period (.) after `Animal`

Comment (linda-seiter) [reply README.md]: fixed

**Review by kcleland0818 — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**